lib/internal/process/execution.js

```diff
@@ -73,11 +86,14 @@ function evalScript(name, body, breakFirstLine, print) {
     global.require = require;
     const { kVmBreakFirstLineSymbol } = global;
     delete global.kVmBreakFirstLineSymbol;
+    const { importModuleDynamically } = module;
+    delete module.importModuleDynamically;
```

Comment:
what's the reason for deleting the `module.importModuleDynamically` after importing it?

Reply:
`module.importModuleDynamically` is not an official API so I didn't want to leak it to the user script.